admin/sql/create_tables.sql

```diff
@@ -42,4 +42,45 @@ CREATE TABLE api_compat.session (
     ts        TIMESTAMP DEFAULT CURRENT_TIMESTAMP
 );
 ALTER TABLE api_compat.session ADD CONSTRAINT session_sid_uniq UNIQUE (sid);
+
+CREATE TABLE statistics.user (
+    user_id                 INTEGER NOT NULL, -- PK and FK to "user".id
+    artists                 JSONB,
+    releases                JSONB,
+    recordings              JSONB,
+    last_updated            TIMESTAMP WITH TIME ZONE
+);
+ALTER TABLE statistics.user ADD CONSTRAINT user_stats_user_id_uniq UNIQUE (user_id);
```

Comment:
We don't need a `UNIQUE` constraint if we also have a `PRIMARY KEY` (a primary key is basically a unique constraint + index)